src/routes/jira/jira-get-connections-backfilStatus.ts

```diff
@@ -0,0 +1,128 @@
+import { NextFunction, Request, Response } from "express";
+import { groupBy, countBy } from "lodash";
+import { RepoSyncState } from "~/src/models/reposyncstate";
+// import { Subscription, TaskStatus } from "~/src/models/subscription";
+import { TaskStatus } from "~/src/models/subscription";
+
+type subscriptionBackfillState = {
+	totalRepos?: number;
+	syncCompleted?: boolean;
+	syncedRepos?: number;
+	backfillSince?: string | null;
+};
+type backFillType = {
+	[key: string]: subscriptionBackfillState;
+};
+
+export const JiraGetConnectionsBackfilStatus = async (
+	req: Request,
+	res: Response,
+	next: NextFunction
+): Promise<void> => {
+	try {
+		const subscriptionIds: number[] = String(req.query.subscriptionIds)
+			.split(",")
+			.map((id) => Number(id))
+			.filter((n) => !!n);
+
+		if (subscriptionIds.length <= 0) {
+			req.log.error("Missing Subscription IDs");
```

Comment:
`req.log.warn` should be enough. As `error` is when the error is cause by our app.

Reply:
Done